paddle/fluid/distributed/collective/gloo_tools.h

```diff
@@ -61,8 +61,6 @@ class SendRecvOptions {
 
   friend void send_recv(SendRecvOptions*);
 };
-
```

Comment:
You do not need to rename this file. Rename it back.

Reply:
Done